Buildings/Experimental/DHC/CentralPlants/Cooling/Subsystems/Examples/CoolingTowerParallel.mo

```diff
@@ -0,0 +1,171 @@
+within Buildings.Experimental.DHC.CentralPlants.Cooling.Subsystems.Examples;
+model CoolingTowerParallel
```

Comment:
Is it possible to extend `Buildings.Fluid.HeatExchangers.CoolingTowers.Examples.BaseClasses.PartialStaticTwoPortCoolingTower` to avoid duplicated code with this example model?

Same goes for `Buildings.Experimental.DHC.CentralPlants.Cooling.Subsystems.Examples.CoolingTowerWithBypass`

Reply:
Note: To use the `PartialStaticTwoPortCoolingTower` base class, we had to modify the base class to remove the `constrainedby` constraint. The modified version is copied below. 

- [ ] This should be noted in the PR to LBNL so that they are aware of this change. 

  ```
replaceable 
    Buildings.Fluid.HeatExchangers.CoolingTowers.BaseClasses.CoolingTower tow(
      redeclare package Medium = Medium_W,
      m_flow_nominal=m_flow_nominal,
      dp_nominal=6000,
      energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
      show_T=true) "Cooling tower"
```